hw2/05-form.html

```diff
@@ -0,0 +1,114 @@
+          </div>
+          <div class="form-group row">
+            <div class="col-form-label">
+              <label for="email" class="fw-bold">Email:</label>
+              <input type="email" class="form-control" id="email" required/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <div class="col-form-label">
+              <label for="registrationStatus" class="col-form-label fw-bold"
+                >Registration Status:</label
+              >
+              <select
+                class="form-select"
+                name="reg-status"
+                id="registrationStatus"
+              >
+                <option value="">Choose an option</option>
+                <option value="registered">Registered</option>
+                <option value="non-registered">Not Registered</option>
+                <option value="waitlist">Waitlist</option>
+              </select>
+            </div>
+          </div>
+          <fieldset class="form-elemnet form-checkboxes">
+            <legend class="col-form-label fw-bold">
+              Have you taken any of following courses?
+            </legend>
+            <br />
+            <input
+              class="form-check-input"
+              type="checkbox"
+              name="courseSelection"
+              id="programmingLanguage"
+            />
+            <label class="form-check-label" for="programmingLanguage"
+              >Programming Languages</label
+            >
+            <br />
+            <input
+              class="form-check-input"
+              type="checkbox"
+              name="courseSelection"
+              id="OperatingSystem"
+            />
+            <label class="form-check-label" for="OperatingSystem"
+              >Operating System</label
+            >
+            <br />
+            <input
+              class="form-check-input"
+              type="checkbox"
+              id="FullStack"
+              name="courseSelection"
+            />
+            <label class="form-check-label" for="FullStack"
+              >Full Stack Web Developement</label
+            >
+          </fieldset>
+          <div class="form-group row">
+            <label for="comment" class="col-form-label fw-bold"
+              >Anything else:</label
+            >
+            <br />
+            <textarea class="form-group" name="comment" id="comment">
+            </textarea>
+          </div>
+          <br />
+          <div>
+            <input
+              type="button"
+              class="btn btn-primary flex-fill"
+              id="Submit"
+              value="Submit"
+              style="width: 50%; float: left"
+              onclick="submitForm()"
+            />
+            <input
+              type="button"
+              class="btn btn-secondary flex-fill"
+              id="Reset"
+              value="Reset"
+              style="width: 50%; float: right"
+              onclick="resetForm()"
```

Comment:
Do not use in-line JavaScript in the HTML file, all the JS code (and function calls), should be in a separate JS file.